### ISSUE TITLE
Memoized data and assets do not support a list of assets

### ISSUE THREAD

**Issue by mtalexan** [mtalexan/VersionOne.SDK.Python#23]
The memoized and custom constructed class is a V1Meta class object.  This can potentially be a list if multiple results are returned from a query though.  The support for more than one item being returned seems to be an update after the fact rather than initially supported by the design.

Split the V1Meta from an Asset class object so the query results can still be memoized by V1Meta, but a consistent interface to the asset objects can be provided wherein it's always a list of Asset objects that contain the attrs.

**Comment by fahrenberger** [comment]
I agree. This has already bit me. If I request the TaggedWith property, V1Meta (in line 223) will only return the first tag of a Story/Epic. This is an actual issue that may cause people headaches. Maybe put a warning in the documentation?
